aws/templates/common.ftl

```diff
@@ -1767,6 +1767,92 @@
 
 [/#function]
 
+[#function getLBLink occurrence port ]
```

Comment:
Is this being used by getContainers as well?  If adding a generic function, shoul dupdate its use in the other places its needed to make sure the function signature is reusable.

Reply:
Added to the containers function as well.